### ISSUE TITLE
Cover page not updated on backend switch

### ISSUE THREAD

**Issue by dikonov** [andywuest/harbour-watchlist#44]
Way to to reproduce:
1. Select Euroinvestor, add some stocks, fetch data.
2. Switch to MOEX, optionally add similar stocks.
3. Leave the app and look at the cover.
The info shown by the cover is wrong. It gets normal after manual data refresh
It seems that clearing of the DB does not affect the cover/does not fully reset data in memory.

**Comment by andywuest** [comment]
Yes - i did not test this use case. I will fix it in the next release.
